packages/auth/src/AuthService.ts

```diff
@@ -68,4 +68,22 @@ export class AuthService {
   public extract(): KeycloakInstance {
     return this.auth;
   }
+
+  /**
+   * Check it the user has a specified realm role
+   */
+  public hasRealmRole(role: string): boolean{
+    return this.auth.hasRealmRole(role);
+  }
+
+
+  /**
+   * Return the users realm level roles
+   */
+  public getRealmRoles():string[]{
+    if (this.auth != null && this.auth.realmAccess != null && this.auth.realmAccess.roles != null) {
```

Comment:
Better check whether the value is falsy instead:
`if (this.auth && this.auth.realmAccess && this.auth.realmAccess.roles) ... `